docs/source/data/fetching-rest.mdx

```diff
@@ -89,75 +126,88 @@ const resolvers = {
+You can pass your `RESTDataSource` subclasses a cache (such as Apollo Server's default cache) to store the results of past fetches:
 
-### Using an external cache backend
+<MultiCodeBlock>
 
-When running multiple instances of your server, you should use a shared cache backend. This enables one server instance to use the cached result from _another_ instance.
+```ts
+import { RESTDataSource } from '@apollo/datasource-rest';
+// KeyValueCache is the type of Apollo server's default cache
+import { KeyValueCache } from '@apollo/utils.keyvaluecache';
+import { ApolloServer } from '@apollo/server';
+import { startStandaloneServer } from '@apollo/server/standalone';
 
-Apollo Server supports using [Memcached](https://memcached.org/), [Redis](https://redis.io/), or other cache backends via the [`keyv`](https://www.npmjs.com/package/keyv) package. For examples, see [Configuring external caching](../performance/cache-backends#configuring-external-caching).
+class MoviesAPI extends RESTDataSource {
+  baseURL = 'https://movies-api.example.com/';
 
-You can also choose to implement your own cache backend. For more information, see [Implementing your own cache backend](../performance/cache-backends#implementing-your-own-cache-backend).
+  //highlight-start
+  constructor(options: {cache: KeyValueCache }) { 
+    super(options); // this should send our server's `cache` through
+  }
+  //highlight-end
 
-## `RESTDataSource` reference
+  // data fetching methods, etc.
+}
 
-The `RESTDataSource` abstract class helps you fetch data from REST APIs. Your server defines a separate subclass of `RESTDataSource` for each REST API it communicates with.
+interface ContextValue {
+  dataSources: {
+    moviesAPI: MoviesAPI;
+  };
+}
 
-To get started, install the `apollo-datasource-rest` package:
+const server = new ApolloServer<ContextValue>({
+  typeDefs,
+  resolvers,
+});
 
-```bash
-npm install apollo-datasource-rest
+const { url } = await startStandaloneServer(server, {
+  context: async ({ req }) => {
+    const { cache } = server;
+    return {
+      token,
+      //highlight-start
+      dataSources: {
+        moviesAPI: new MoviesAPI({ cache, token }),
+      },
+      //highlight-end
+    };
+  },
+});
 ```
 
-You then extend the `RESTDataSource` class and implement whatever data-fetching methods your resolvers need. These methods can use built-in convenience methods (like `get` and `post`) to perform HTTP requests, helping you add query parameters, parse JSON results, and handle errors.
+</MultiCodeBlock>
 
-### Example
+If you want Apollo Server to use another cache backend, you can provide its constructor a _different_ cache object that implements the [`KeyValueCache` interface](https://github.com/apollographql/apollo-utils/tree/main/packages/keyValueCache#keyvaluecache-interface):
```

Comment:
Custom cache for Apollo Server / external caches should definitely go on a different page from RESTDataSource stuff.

Reply:
Great point; I added a callout with a link to the Configuring Cache backends article. :)